fcrepo-postgres.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.6"
+
+networks:
+  appnetwork:
+    driver: bridge
+
+services:
+  fcrepo:
+    container_name: fcrepo
+    build: 
+      context: ./docker/services/fcrepo
+      args:
+        FEDORA_VERSION: 4.7.5
+        FEDORA_TAG: 4.7.5
+        ModeshapeConfig: jdbc-postgresql
+        JDBCConfig: -Dfcrepo.postgresql.username=fcrepoadmin -Dfcrepo.postgresql.password=fcrepoadminpw -Dfcrepo.postgresql.host=postgres.docker.local -Dfcrepo.postgresql.port=5432 
```

Comment:
Like the comment for `fcrepo-mysql.yml`, shouldn't these settings try and reference the environment variables for the database service, where possible, in order to ensure they match up when changed (and meaning they only need to be changed in one place)?

Reply:
Same as above.